cernopendata/config.py

```diff
@@ -328,6 +360,7 @@
             file_type=terms_filter('distribution.formats.keyword'),
             collections=terms_filter('collections.keyword'),
             availability=terms_filter('distribution.availability.keyword'),
+            event_number= None,
```

Comment:
So with the above change, we should just import the default `range_filter` and use it here:

```python
from invenio_records_rest.facets import range_filter
...
            event_number=reange_filter('distribution.number_events') 
```
Can you try that and share the results? Thank you :)